editor/import/resource_importer_wav.cpp

```diff
@@ -112,7 +112,13 @@ Error ResourceImporterWAV::import(const String &p_source_file, const String &p_s
 	}
 
 	/* GET FILESIZE */
-	file->get_32(); // filesize
+	uint32_t file_size_header = file->get_32() + 8; // Offset 8 byte
```

Comment:
That isn't an offset, isn't it? You just added `8` to the value.

Reply:
It's the file size in header is 8 bytes less than the actual file size. It is a little confusing, I will change this.